src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests.cs

```diff
@@ -59,8 +59,8 @@ internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProvider
             bool outOfProcess)
         {
             await TestInRegularAndScript1Async(
-                initialMarkup, expectedMarkup, index, priority,
-                parameters: new TestParameters(options: options, fixProviderData: outOfProcess));
+                initialMarkup, expectedMarkup, index,
+                parameters: new TestParameters(options: options, fixProviderData: outOfProcess, priority: priority));
```

Comment:
why this change?

Reply:
Let me try to pull this out into a separate PR.

Reply:
https://github.com/dotnet/roslyn/pull/41635